CodeStarterCamp_Week2/step 3_storeLottoNumbers.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  step 3_storeLottoNumbers.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by Minseong Kang on 2022/02/23.
+//
+
+import Foundation
+
+var numbersByRounds : [Int: Set<Int>] = [:]
+var round = 1
+
+
+func storeWinningNumbersByRounds(roundNumber: Int, winningNumbers: Set<Int>) {
+    numbersByRounds.updateValue(winningNumbers, forKey: round)
+    round = round + 1
+}
+
+
+func printWinningNumbersByRounds(roundNumber: Int) {
+    guard numbersByRounds[roundNumber] != nil else {
+        printUnvalidRoundMessage(inputRound: roundNumber)
+        return
+    }
+    print("\(roundNumber)회차의 로또 당첨 번호는 \(convertingIntSetToStringArray(numbersByRounds[roundNumber]!).joined(separator: ", ")) 입니다.")
+}
+
+
+func convertingIntSetToStringArray(_ intSet: Set<Int>) -> Array<String> {
```

Comment:
[API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/)에서 Strive for Fluent Usage
 -> Name functions and methods according to their side-effects 에서 More details를 누르면 나오는 아래의 항목을 봤어요. 코드 예시에서 -ing 를 붙이는 걸 보고 따라해봤는데, 지금 다시 읽어보니 잘 이해하지 못한 상태로 제 마음대로 생각하고 받아들인 것 같아요. 빠르게 읽으면서 '아 뭔가 어법상 -ing 혹은 -ed 형으로 쓰는 게 자연스러우면 그래도 되는구나!' 정도로만 생각했던 것 같아요 🥲 역시 모든 문서는 꼼꼼하게 잘 읽어야합니다..! 😥

```
Prefer to name the nonmutating variant using the verb’s past [participle](https://en.wikipedia.org/wiki/Participle) (usually appending “ed”):

/// Reverses `self` in-place.
mutating func reverse()

/// Returns a reversed copy of `self`.
func reversed() -> Self
...
x.reverse()
let y = x.reversed()
When adding “ed” is not grammatical because the verb has a direct object, name the nonmutating variant using the verb’s present [participle](https://en.wikipedia.org/wiki/Participle), by appending “ing.”

/// Strips all the newlines from `self`
mutating func stripNewlines()

/// Returns a copy of `self` with all the newlines stripped.
func strippingNewlines() -> String
...
s.stripNewlines()
let oneLine = t.strippingNewlines()
```